appserver/monitoring-console/core/src/test/java/fish/payara/monitoring/model/SeriesTest.java

```diff
@@ -94,4 +95,23 @@ public void tagAssignmentIsSpecialCharacter() {
     public void wildCardIsSpecialCharacter() {
         assertTrue(Series.isSpecialTagCharacter('*'));
     }
+
+    @Test
+    public void illegalFormat() {
+        try {
+            assertNotNull(new Series("ns: system. cpu.load"));
+        } catch (IllegalArgumentException e) {
+            assertEquals("Malformed series key, `:` missing or misplaced in ns: system. cpu.load", e.getMessage());
+        }
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void nullArgument() {
+        assertNotNull(new Series(null));
```

Comment:
here is correct to add fail(message] after the new Series and remove the assertNotNull which will never be called

Reply:
I tend to use the `assertNotNull` as a way to do something with the outcome should it not throw an exception so static analysis does not throw out the statement and the exception never occurs for that reason. Maybe a message could make that clear.